libraries/Microsoft.Bot.Builder.Core/TurnContext.cs

```diff
@@ -190,7 +190,7 @@ public bool Responded
             if (activity == null)
                 throw new ArgumentNullException(nameof(activity));
 
-            ResourceResponse[] responses = await SendActivities(new IActivity[] { activity });
+            ResourceResponse[] responses = await SendActivities(new [] { activity });
```

Comment:
I've never seen that syntax before. Nice.

Reply:
Compiler knows what you're asking for, just reduces the noise. :)